internal/assets/contents/usage.tpl

```diff
@@ -11,11 +11,11 @@ Usage:
 Aliases:
     {{.Cobra.NameAndAliases}}
 {{- end}}
-{{- if .Cobra.HasExample}}
 
 Examples:
-    {{.Cobra.Example}}
-{{- end}}
+{{- range  .Cmd.Examples }} 
+    {{ . -}}
+{{ end }}
```

Comment:
I think this also needs a `{{- if .Cmd.Examples }}`

Reply:
Oh right. Good catch.